src/engines/html5/html5.js

```diff
@@ -823,6 +827,9 @@ export default class Html5 extends FakeEventTarget implements IEngine {
   _init(source: PKMediaSourceObject, config: Object): void {
     this._config = config;
     this._loadMediaSourceAdapter(source);
+    if (this._mediaSourceAdapter) {
```

Comment:
Move this line to be inside ```_loadMediaSourceAdapter``` since you always need it incase MSE is exists.
Then you won't need the condition.